Library/Homebrew/service.rb

```diff
@@ -355,6 +357,23 @@ def command
       @run&.map(&:to_s)
     end
 
+    sig { params(value: T.nilable(Integer)).returns(T.nilable(Integer)) }
+    def nice(value = nil)
+      case T.unsafe(value)
+      when nil
+        @nice
+      when Integer
+        if NICE_RANGE.cover?(value)
+          @nice = value
+        else
+          raise TypeError,
+                "Service#nice value should be between #{NICE_RANGE}"
+        end
```

Comment:
```suggestion
        raise TypeError, "Service#nice value should be in #{NICE_RANGE}" unless NICE_RANGE.cover?(value)

        @nice = value
```
As a slightly improved error message and slightly more idiomatic ruby